### DISCUSSION TITLE
request hanging for minutes

### DISCUSSION THREAD

**Discussion by charlower** [ashleykleynhans/runpod-worker-a1111#11] [Q&A] [answered]
Hi Ashley, thanks for building this. I'm using your template on Runpod and sending the example request in the documentation "txt2img.md", however the request appears to be hanging for a long time. At the time of writing this its been 10 minutes. Here is an example of the server logs:

<img width="1452" alt="Screen Shot 2023-11-29 at 5 21 07 pm" src="https://github.com/ashleykleynhans/runpod-worker-a1111/assets/7565117/46fb9d5b-5a57-4543-9c10-53bdbb4d9b16">

It just seems to be stuck on "Service not ready yet. Retrying...". Any help would be greatly appreciated.

**Comment by ashleykleynhans** [comment, marked as answer]
You didn't follow the instructions properly, do it again.

**Reply by charlower** [reply]
you sure somethings not broken?

**Reply by ashleykleynhans** [reply]
Nothing is broken, you did something wrong.

**Reply by charlower** [reply]
yes sir thank you
